docs/using.md

```diff
@@ -60,6 +60,21 @@ Another common example might be the URL of the ISO to be used. Usually you speci
 _iso-url_ of the [start](./docs/minishift_start.md) command. Applying the rules from above, you can
 also specify this URL by setting the environment variable _MINISHIFT_ISO_URL_.
 
+Minishift also maintain a config file (_~/.minishift/config/config.json_) which can be used to set some
```

Comment:
On a side note, right now, _config.json_ does not get generated unless you execute a _set_ command. This is a bit confusing for people looking for this file. So either we need to document that this file does not exist unless you add some custom configuration or we make sure that an empty config file gets generated in case it does not exist.